rule-packs/gcp.json

```diff
@@ -0,0 +1,134 @@
+[
+    {
+      "name": "gcp-personal-account-access",
+      "description": "Finds login credentials tied to personal accounts (as opposed to corporate credentials).",
+      "queries": [
+        {
+          "name": "query0",
+          "query": "find google_user with email !$='@{{domain}'",
+          "version": "v1"
+        }
+      ]
+    },
+    {
+      "name": "gcp-managed-service-account-keys",
+      "description": "Finds user managed serice accounts with user managed keys (as opposed to system managed keys).",
```

Comment:
Is there going to be an effort to describe rules consistently? I bumped around a few other files in this `rule-packs` directory and see that many do not start with `Finds ...` (some are `Identitifies...`, some `Find....`, some neither of those two things).

Reply:
To clarify, I like that this file uses `Finds...` consistently!

Reply:
We will definitely be adding some consistency going forward, this is a good one!